frontend/README.md

```diff
@@ -59,3 +59,11 @@ to
 axios.defaults.withCredentials = true;
 ```
 This should keep you out of CORS troubles
+
+## Debugging the application
+
+* Follow the instructions in the `Build and run the frontend locally` to start application

```

Comment:
You can hyperlink to another section

https://gist.github.com/asabaylus/3071099

Reply:
Done.